charts/gateway/templates/config.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: gateway-config
+data:
+  gateway.conf: |-
+    server {
+      error_log  /var/log/nginx/error.log info;
+      listen       9443 ssl;
```

Comment:
Why did this change to 9443? Not a big deal, but it is one of those subtle changes that is not backwards compatible with devices who already have a sota.toml file deployed.

Reply:
It was copied from another template. I'll change it back to 8443.